src/front/spv.rs

```diff
@@ -10,11 +10,16 @@ TODO: would be nice to find ways that avoid looking up as much
 
 !*/
 
+use std::convert::TryFrom;
+
+use crate::front::spv_dag::*;
```

Comment:
we really need to turn SPV front-end into a real module, i.e. have `src/front/spv/mod.rs` for this file, then `src/front/spv/dag.rs` for your new one

Reply:
Discussed in the Element.